src/python/scripts/dx-docker

```diff
@@ -210,9 +210,11 @@ def run(args):
 
     container_cmd = entrypoint + cmd
 
-    workdir = args.workdir
+    workdir = "/"
     if 'workingDirectory' in imagemeta['app']:
        workdir = imagemeta['app']['workingDirectory']
+    if args.workdir:
```

Comment:
Thanks for catching this! Looks like your comments are spot on. I am currently testing out a fix and adding a few useful test cases. Also outputting metadata for the working directory for future reference :).